### ISSUE TITLE
change the default of nextDayThreshold to 00:00

### ISSUE THREAD

**Issue by lolo49lala** [fullcalendar/fullcalendar#2902]
## Rendering problem for multiple-day-events on month-view when event is ending before 09am

Hello folk,

I have found the following bug in fullcalender 2.4.0.
I modified a copy of default.html from demo-package and add some own events. 

(see following code)

``` javascript
$('#calendar').fullCalendar({
            editable: true,
            //timezone:'Europe/Berlin',
            eventLimit: true, // allow "more" link when too many events
            events: [
                {
                    title: 'Meeting',
                    start: '2015-10-12T10:30:00',
                    end: '2015-10-13T12:30:00' // <<<correct
                },
                {
                    title: 'TEST from 22-06',
                    start: '2015-10-18T22:00:00',
                    end: '2015-10-19T06:00:00'   // <<<bug
                },
                {
                    title: 'TEST from 12-08:59',
                    start: '2015-10-20T12:00:00',
                    end: '2015-10-21T08:59:00' // <<<bug
                },

                {
                    title: 'TEST from 22-09',
                    start: '2015-10-23T22:00:00',
                    end: '2015-10-24T09:00:00'  // <<<correct
                },
                {
                    title: 'XXX22-20',
                    start: '2015-10-27T22:00:00',
                    end: '2015-10-29T20:00:00' // <<<correct
                }
            ]
        });
```

Can anyone analyse the problem.

Have many thanks in advance.

Best regards
![screenshot](https://cloud.githubusercontent.com/assets/8592414/10552947/af2ae34e-7457-11e5-8c64-be044178eb33.PNG)

**Comment by arshaw** [comment]
this is likely due to nextDayThreshold:
http://fullcalendar.io/docs/event_rendering/nextDayThreshold/

if you set it to `'00:00'` it will probably work

im keeping this ticket open for discussion about whether we should change the default to 00:00 in the future, for a major version upgrade.

**Comment by philipstratford** [comment]
It seems to me that a default value for the nextDayThreshold propery of `00:00` would be more intuitive. It can always be changed if required.

**Comment by lolo49lala** [comment]
@althaus 
Because the default-value is illogical I run in problems where I lost 1 day to analyse this. Your API is very good, but this was a big fault.

Thank you for posting the documentation-link, so  I can remove my workaround in month-view. Please change the default-value so soon as possible.

**Comment by althaus** [comment]
Wild guess: @lolo49lala meant @arshaw and not me. :smile:

**Comment by lolo49lala** [comment]
Iam sorry, I mean @arshaw.
Auto-Complete is good, when I tying the right letters :)

**Comment by arshaw** [comment]
i agree, this should be changed back to `00:00`. will be a breaking and surprising change to some, so will wait for the next major release.

**Comment by guilhermedecampo** [comment]
👍  also agree with back to 00:00. Spent some good hours to find this issue!

**Comment by arshaw** [comment]
i originally set to 09:00 because that's how Google Calendar decided to render things. it seems however that Google Calendar changed their value to `05:00`

regardless, I still think this should be reset to 00:00

**Comment by davidwilcox** [comment]
+1. I ran into this the other day and was very confused until I just happened upon a setting to change it back to 00:00.

**Comment by kkazants** [comment]
I set the nextDayThreshold to '00:00' and noticed weird actions. It seems like it is inconsistent, but if you have an event that has a start and end date as: '2017-02-14T22:11:00', the event ends up looking like it ends the next day:
![image](https://cloud.githubusercontent.com/assets/10411394/24810334/155b86b0-1b90-11e7-96c6-d8d418fddd96.png)
Have used default.html for testing. Added following lines to events array:
,{title: 'test1',start: '2017-02-14T22:11:00',end: '2017-02-14T22:11:00'}

Any suggestions?

**Comment by rmc0207** [comment]
Also have the same problem, example code - http://stackoverflow.com/questions/43367984/fullcalendar-next-day-threshold-not-working-on-listweek-view

**Comment by kkazants** [comment]
Another question concerning the nextDayThreshold, does it set it to 00:00 UTC time or local time? Thanks.

**Comment by arshaw** [comment]
if the calendar's [timezone](https://fullcalendar.io/docs/timezone/timezone/) is `'local'`, it will be local if the timezone is `'UTC'`, it will be UTC, if the timezone if `false` or custom, it will be ambiguously zoned

**Comment by mtilokani** [comment]
@arshaw @kkazants 
I am facing same issue when start and end datetime are same, then it shows a 2 hours of event reservation on fullcalendar, and actually it should not show as start and end are same so it should neglect.
Please help.

**Comment by mtilokani** [comment]
@arshaw @kkazants 

Here jsfiddle http://jsfiddle.net/a7mad24/jso51pm6/ where start and end are same.
And it shows 2 hours as reserved from 7am to 9am, why is it like so? even if date and time are same.
To check this you need to set your system timezone.
Please help

Thanks

**Comment by arshaw** [comment]
This issue has been addressed in the latest alpha release (4.0.0-alpha.2)
[See the blog post &raquo;](https://fullcalendar.io/blog/2018/10/release-v4-alpha2)

**Comment by JorgeGante** [comment]
I´m having problems with nextDayThreshold too. Even when I´m setting it to "08:00:00" FullCalendar is duplicating events that finish at "07:00:00" on the 1st of a month, displaying it with a left arrow in timelineMonth, showing the same event in two months.

https://jsfiddle.net/q2fk57nb/

**Comment by arshaw** [comment]
@JorgeGante , I believe your issue is a duplicate of:
#3943 
&uarr;
Could you please visit the link, **give a** 👍  **to vote** ([?](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments)), press the 🔈**Subscribe** button to receive updates?

if so, it is likely fixed with the release of v4.0.0-alpha.3. can you please try it? if the bug persists, please comment on #3943, or if it's a new issue, post a new issue. please don't post on here.

Anyway, I'm closing this issue to signify that it's implemented (since v4-alpha2). I'll comment on this thread again when v4 is officially released.

**Comment by arshaw** [comment]
[V4 was officially released a few days ago!](https://fullcalendar.io/blog/2019/03/v4-officially-released)
